packages/model/src/team.ts

```diff
@@ -65,14 +65,22 @@ export type TeamCreateDataObject = {
   tools?: TeamTool[];
 };
 
-export type ListTeamDataObject = ListResponse<TeamDataObject>;
-
 export type TeamResponse = TeamDataObject;
 
-export type ListTeamResponse = ListResponse<TeamResponse>;
+export type FetchTeamsOptions = FetchOptions;
+
+export type TeamListItemDataObject = Pick<
+  TeamDataObject,
+  | 'id'
+  | 'displayName'
+  | 'inactiveSince'
+  | 'projectTitle'
+  | 'expertiseAndResourceTags'
+  | 'labCount'
+> & { members: number };
```

Comment:
`members` implies a list of members (which it was previously), not a count. If we do not need a list that's fine but rename the property to, say, `memberCount` or something like that?